include/sdbus-c++/IProxy.h

```diff
@@ -65,6 +65,11 @@ namespace sdbus {
     public:
         virtual ~IProxy() = default;
 
+        /*!
+         * @brief return object path of the underlying DBus object
+         */
+        virtual const std::string& getObjectPath() const = 0;
```

Comment:
Ditto, moving to the bottom of the class...

Reply:
Done